### ISSUE TITLE
Providing both Relay and Parachain chainspec fails

### ISSUE THREAD

**Issue by samelamin** [paritytech/zombienet#701]
When specifying both a chainspec for relay and parachain nodes fails with the below message

```
UnhandledRejection:  	 Error: Cannot read /var/folders/9b/7hmm42694c1flq8nqw7z32_h0000gn/T/zombie-46614f0a07019374b693afd445c0773f_-66593-O07QqV0Zr5iq/2000/genesis-wasm: Error: ENOENT: no such file or directory, open '/var/folders/9b/7hmm42694c1flq8nqw7z32_h0000gn/T/zombie-46614f0a07019374b693afd445c0773f_-66593-O07QqV0Zr5iq/2000/genesis-wasm'

```

**Comment by samelamin** [comment]
Hi @pepoviola 

I added the feature and toml file in this pr https://github.com/paritytech/cumulus/pull/1909/files#diff-369f3294111b6bc34fe0a22c0bab19f71aecd065463192bc2c1431f61c273936

**Comment by pepoviola** [comment]
Hi @samelamin this was fixed by your pr right? #673?
Thanks!
